### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by SimonSK**
- added log messages for unexpected/unknown devp2p packets
- gofmt
won't affect the currently running case-study

### CODE REVIEW

File: log/format.go [hunk 1]
```diff
@@ -14,10 +14,11 @@ import (
 )
 
 const (
-	timeFormat     = "2006-01-02T15:04:05.999999999-0700"
-	termTimeFormat = "01-02|15:04:05"
-	floatFormat    = 'f'
-	termMsgJust    = 40
+	timeFormat        = "2006-01-02T15:04:05-0700"
+	termTimeFormat    = "01-02|15:04:05"
+	newTermTimeFormat = "2006-01-02T15:04:05.999999999-0700"
```
Comment (zzma) [inline log/format.go]: Can we rename this to something more semantically descriptive? like `nanoTermTimeFormat`

also, just want to verify that timeFormat and termTimeFormat still being used

**Review by zzma — APPROVED** [review]
minor comment, otherwise LGTM

File: log/format.go (hunk 1 above)
Comment (SimonSK) [reply log/format.go]: timeFormat is used somewhere else. `nanoTermTimeFormat` replaces `termTimeFormat`. The IDE doesn't find any usage of it.

**Review by sid1602 — APPROVED** [review]
Looks good.

File: log/format.go (hunk 1 above)
Comment (zzma) [reply log/format.go]: gotcha. then I'd say use the name `termTimeFormat` to represent the new time format with nano-seconds.

**Pull request merged**